account_due_list_days_overdue/models/account_move_line.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+# © 2015 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models, _
+import datetime
+from lxml import etree
+
+
+class AccountMoveLine(models.Model):
+    _inherit = 'account.move.line'
+
+    days_overdue = fields.Integer(compute='_compute_days_overdue',
+                                  search='_search_days_overdue',
+                                  string='Days overdue')
+    overdue_term_1 = fields.Float(string='Overdue Term 1',
```

Comment:
I am really fan of this kind of design: I find it not flexible for future use/enhancement. Why not a list (o2m) with limitation to 5 elements?